articles/quickstart/spa/angular-next/02-calling-an-api.md

```diff
@@ -0,0 +1,186 @@
+      scope: "read:current_user",
+
+      // Specify configuration for the interceptor              
+      httpInterceptor: {
+        allowedList: [
+          {
+            // Match any request that starts 'https://${account.namespace}/api/v2/' (note the asterisk)
+            uri: 'https://${account.namespace}/api/v2/*'
+            tokenOptions: {
+              // The attached token should target this audience
+              audience: 'https://${account.namespace}/api/v2/',
+
+              // The attached token should have these scopes
+              scope: 'read:current_user'
+            }
+          }
+        ]
+      }
+    }),
+  ],
+  providers: [
+    // Register the `AuthHttpInterceptor` to automatically attach the 'Authorization' header
+    { provide: HTTP_INTERCEPTORS, useClass: AuthHttpInterceptor, multi: true },
+  ],
+  bootstrap: [AppComponent],
+})
+```
+
+Auth0 uses the value of the `audience` prop to determine which resource server (API) the user is authorizing your Angular application to access.
+
+Notice the `httpInterceptor` configuration, which specifies:
+
+* Any request where the URL starts with `https://${account.namespace}/api/v2/` should have an access token attached
+* If the URL is matched, the attached access token should have an audience value of `https://${account.namespace}/api/v2/` and the `read:current_user` scope;
+
+A more detailed run-down of the available configuration options for the HTTP interceptor is available in [the repository docs](https://github.com/auth0/auth0-angular#call-an-api).
+
+:::note
+In the case of the Auth0 Management API, the audience is `https://${account.namespace}/api/v2/`. In the case of your APIs, you create an _Identifier_ value that serves as the _Audience_ value whenever you [set up an API](https://auth0.com/docs/getting-started/set-up-api) with Auth0.
+:::
+
+The actions that your Angular application can perform on the API depend on the [scopes](https://auth0.com/docs/scopes/current) that your access token contains, which you define as the value of `scope`. Your Angular application will request authorization from the user to access the requested scopes, and the user will approve or deny the request.
+
+:::note
+In the case of the Auth0 Management API, the `read:current_user` and `update:current_user_metadata` scopes let you get an access token that can retrieve user details and update the user's information. In the case of your APIs, you'll define custom [API scopes](https://auth0.com/docs/scopes/current/api-scopes) to implement access control, and you'll identify them in the calls that your client applications make to that API.
+:::
+
+## Make an API Call
+
+With your app module configured with the HTTP interceptor from the Angular SDK, calls you make to the Auth0 Management API will have a valid access token specified in the `Authorization` header. Let's use this as an example for showing user metadata.
```

Comment:
we can't know if it's valid, the API will decide that :) we can only know whether it's expired or not. We should briefly mention that we also refresh tokens automatically before use, when necessary, otherwise some developers will try to do it themselves and dig around for the RT